src/account/operations/helpers/time.rs

```diff
@@ -134,3 +136,65 @@ pub(crate) fn can_output_be_unlocked_now(
         false
     }
 }
+
+// Check if an output can be unlocked by one of the account addresses at the current time/milestone index and at any
+// point in the future
+pub(crate) fn can_output_be_unlocked_forever_from_now_on(
+    // We use the addresses with unspent outputs, because other addresses of the account without unspent outputs can't
+    // be related to this output
+    account_addresses: &[AddressWithUnspentOutputs],
+    output_data: &OutputData,
+    current_time: u32,
+    current_milestone: u32,
+) -> bool {
+    let mut can_be_unlocked_forever = Vec::new();
```

Comment:
Why not just return when conditions are false ? If you prefer returning at the very end, I would use a single bool that I change state instead of a whole vector

Reply:
Good catch, I copied it from another place where I would then for the expiration check the address in the address unlock condition or the one in the expiration unlock condition, depending on if it's expired or not, so I couldn't return right away
But for this case here we can do it, because we're only interested in the expired state